regression/ansi-c/static3/test.desc

```diff
@@ -2,5 +2,7 @@ CORE
 main.c
 main2.c --function foo
 ^main symbol `foo' is ambiguous$
+^EXIT=1$
```

Comment:
I believe this needs to be `^EXIT=(64|1)$` to cope with Windows.

Reply:
Do you know why?

Reply:
Ah never mind I got it, goto-cc vs. goto-cl